apollo-converters/moshi/src/main/java/com/apollostack/android/PostBodyJsonAdapter.java

```diff
@@ -0,0 +1,34 @@
+package com.apollostack.android;
+
+import com.apollostack.api.GraphQLOperation;
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.JsonReader;
+import com.squareup.moshi.JsonWriter;
+import com.squareup.moshi.Moshi;
+
+import java.io.IOException;
+
+class PostBodyJsonAdapter<T extends GraphQLOperation.Variables> extends JsonAdapter<PostBody<T>> {
```

Comment:
Just curious why do we need to create custom moshi adapter, can't we just use the standard moshi?